UI/src/view/navigationpanel/searchBox.js

```diff
@@ -47,29 +47,11 @@
 
       var showDialog = function(message) {
         instructionsPopup.show(message, 3000);
+        jQuery('.spinner-overlay-search').remove();
+        jQuery('#search-results').parent().hide();
       };
       locationSearch.search(coordinatesText.val()).then(function(results) {
         populateSearchResults(results);
```

Comment:
Please, check if this code is not even necessary. I think when we only have one result, the map should automatically move